components/dropdown/dropdown.js

```diff
@@ -22,6 +22,11 @@ class Dropdown extends DropdownOpenerMixin(LitElement) {
 	 */
 	getOpenerElement() {
 		if (!this.shadowRoot) return undefined;
+		if (this.slottedOpener)  {
```

Comment:
Instead of introducing an attribute, we could also first check the default slot and if it returns nothing, look in the `opener` slot? Shouldn't affect performance too much. 🤷

Reply:
The default slot will always have content (ie. the dropdown-content). Did you mean to check the default slot and do the filtering for the opener as well?